tests/tensor_ops/test_getitem.py

```diff
@@ -220,3 +241,27 @@ def test_getitem_basic_w_adv_fwdprop():
 )
 def test_getitem_basic_w_adv_bkprop():
     pass
+
+
+@fwdprop_test_factory(
+    mygrad_func=get_item,
+    true_func=get_item,
+    num_arrays=1,
+    index_to_arr_shapes={0: hnp.array_shapes(max_side=4, max_dims=8)},
+    kwargs=dict(index=arb_index_wrap),
+)
+def test_getitem_arbitraryindex_fwdprop():
+    pass
+
+
+@settings(deadline=None)
```

Comment:
This is a powerful test - it is worth letting hypothesis do a lot of searching for edge cases
```suggestion
@settings(deadline=None, max_examples=500)
```